server/karaf/features/pom.xml

```diff
@@ -3,8 +3,8 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.apache.james</groupId>
         <artifactId>james-server</artifactId>
+        <groupId>org.apache.james</groupId>
```

Comment:
No it is not, but such weird stuff is preserved by the plugin... So I would argue it is *orthogonal to this PR*.